src/encoded/static/components/matrix_reference_epigenome.js

```diff
@@ -350,10 +350,16 @@ const convertReferenceEpigenomeToDataTable = (context, expandedRowCategories, ex
 const MatrixHeader = ({ context, showProjects, project }, reactContext) => {
     const visualizeDisabledTitle = context.total > MATRIX_VISUALIZE_LIMIT ? `Filter to ${MATRIX_VISUALIZE_LIMIT} to visualize` : '';
 
-    const projectSelect = (e) => {
+    const projectSelect = (e, baseUrl) => {
         const selectedProject = e.target.value;
         const awardRfa = selectedProject === 'All' ? '' : `&award.rfa${selectedProject === 'Roadmap' ? '=' : '!='}Roadmap`;
-        const link = `/reference-epigenome-matrix/?type=Experiment&related_series.@type=ReferenceEpigenome&replicates.library.biosample.donor.organism.scientific_name=Homo+sapiens${awardRfa}`;
+        const query = new QueryString(baseUrl);
```

Comment:
Technically you should only pass the query-string part of the URL to `new QueryString` but it works out here because you’re only manipulating the award.rfa part. If you had tried to manipulate the type=Experiment part, it wouldn’t have worked. So let’s let it go since this will work by avoiding the part where it doesn’t.